<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ISearchDialog</class>
 <widget class="QDialog" name="ISearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SearchDialog</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="mpWhatToSearchGroupBox">
     <property name="title">
      <string>Search for ...</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLineEdit" name="mpSearchLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="mpGoButton">
        <property name="text">
         <string>&amp;Go</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QCheckBox" name="mpCaseSensitiveCheckBox">
       <property name="text">
        <string>&amp;Case Sensitive</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mpMoreButton">
       <property name="text">
        <string>More &gt;&gt;&gt;</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="mpWhereToSearchGroupBox">
     <property name="title">
      <string>Search within ...</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QRadioButton" name="mpSearchAllEntriesRadioButton">
        <property name="text">
         <string>... all Entries</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mpSearchCurrentEntryRecursivelyRadioButton">
        <property name="text">
         <string>... current Entry and Children</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mpSearchCurrentEntryOnlyRadioButton">
        <property name="text">
         <string>... current Entry only</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>524</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTreeWidget" name="mpResultTreeWidget">
     <column>
      <property name="text">
       <string>Entry Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Entry Content</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="mpStatusLine">
     <property name="text">
      <string notr="true">StatusLine</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mpSearchLineEdit</tabstop>
  <tabstop>mpGoButton</tabstop>
  <tabstop>mpCaseSensitiveCheckBox</tabstop>
  <tabstop>mpResultTreeWidget</tabstop>
  <tabstop>mpSearchAllEntriesRadioButton</tabstop>
  <tabstop>mpSearchCurrentEntryRecursivelyRadioButton</tabstop>
  <tabstop>mpSearchCurrentEntryOnlyRadioButton</tabstop>
  <tabstop>mpMoreButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mpMoreButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>mpWhereToSearchGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
